### ISSUE TITLE
hi i have problem- -function open ssl_pkey_free is depraced since php 8.0

### ISSUE THREAD

**Issue by senordelanochexd** [PHPMailer/PHPMailer#3025]
https://github.com/PHPMailer/PHPMailer/blob/5251d331249900a8551442391265a3ec20200c8f/src/PHPMailer.php#L4898

also i cannot remove it because then it has obivously problem on another line -- public function DKIM_HeaderC($signHeader). si is there anyone who know what to do ?

**Comment by XL-2000** [comment]
What is your problem?
If you are running PHP >= 8.0, this function is not called. If you are running PHP < 8.0, it is not deprecated on your system.
So why is this a problem for you?

**Comment by Synchro** [comment]
Indeed, you need to suppress the warning in whatever you have that is complaining about it; it's a false positive.
